framework/templates/ecl-templates-search/ecl-templates-search.config.js

```diff
@@ -1,12 +1,16 @@
 module.exports = {
   title: 'Search Page',
   label: 'Search Page',
-  status: 'wip',
+  status: 'ready',
   preview: '@preview-full-page',
   tags: ['template'],
   context: {
     global: {
       language: 'en',
     },
+    _demo: {
+      scripts:
+        "document.addEventListener('DOMContentLoaded', function () { ECL.initExpandables(); });",
```

Comment:
- regarding the arrow, we don't provide this behavior ("point to the right"). A "collapse button" either shows an "up" arrow or a "down" arrow.
- I will add an example with nested content soon, with indentation ;)
- the `ecl-links` twig template needs to be updated